storage/rocksdb/ha_rocksdb.cc

```diff
@@ -4210,7 +4210,6 @@ longlong ha_rocksdb::update_hidden_pk_val() {
 
 /* Get the id of the hidden pk id from m_last_rowkey */
 int ha_rocksdb::read_hidden_pk_id_from_rowkey(longlong *const hidden_pk_id) {
```

Comment:
IMHO we should remove attribute nonnull instead. Its abuses with nullptrs passed will not be caught by compiler nor generated code, leading to nasty "miscompilations", we had two instances of that in the core server. @georgelorchpercona  ?